youki_integration_test/src/command/create.rs

```diff
@@ -0,0 +1,19 @@
+use std::path::PathBuf;
+use std::process::{Command, Stdio};
+
+// TODO Allow to receive arguments.
+// TODO Wrapping up the results
+pub fn exec(project_path: &PathBuf, id: &str) -> bool {
+    let status = Command::new(project_path.join(PathBuf::from("youki")))
```

Comment:
I think `"youki"` is too hard coding, so how about using an environment, e.g. `RUNTIME`
I think the default value should be youki.

Reply:
What would be your thoughts on using cmd args instead of env vars? That way we user won't have to define env vars specifically for this purpose, and it might make more sense to pass those as args to the command, rather than to set them as general env vars.

Reply:
@utamoku What do you think about this?
My feeling is that when comparing `env vars` and `cmd args`, `env vars` is basically unchanged and users are expected to use default values, while `cmd args` is assumed to be specified by users.
In this case, I think `env vars` is fine because it is basically used with `youki`, but `cmd args` may be an option if you are considering using a runtime other than `youki` in the future.